### PR TITLE
Play music

### PR DESCRIPTION
**Pull request by amiraelsayedbadwy**
- [x] GamerTag Page
    - [x] Place audio file locally and play audio automatically on login screen
    - [x] Get music from Linda
    - [x] Use mute and audio on icons, as shown here: https://www.figma.com/file/KuRbzIk6FzNXK5qVanjfr2/IslandLanding?node-id=0%3A1
- [x] In NextPopupViewModel.cs,       else if (GameModel.TakenTime <= 0.1 || GameModel.TakenTime >= -0.1)        {          Seconds = GameModel.MainTime + " seconds, your accuracy is amazing, you must be a skydiver!";        } Tutorial 1 :

### CODE REVIEW

File: IslandLanding/IslandLanding/Converters/ImageConverter.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  {
+
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+      string ButtonImage = "volume_off_24px.png";
+
+      if (value is string)
+      {
+        var imageValue = (string)value;
+        if (!string.IsNullOrEmpty(imageValue))
```
Comment (saamerm) [inline IslandLanding/IslandLanding/Converters/ImageConverter.cs:19]: You can shrink this to one line:
'if (value is string imageValue && !string.IsNullOrEmpty(imageValue))'

File: IslandLanding/IslandLanding/ViewModel/GamerTagViewModel.cs [hunk 2]
```diff
@@ -29,10 +29,10 @@ public string PopupText
     public GamerTagViewModel()
     {
       SaveCommand = new Command(SaveCommandExcute);
+      InfoViewModel.Play();
```
Comment (saamerm) [inline IslandLanding/IslandLanding/ViewModel/GamerTagViewModel.cs:32]: Don't make 'Play()' a static function that's called like this. 
I would recommend creating an AudioService that controls pause and play and also stores the state of whether the audio is playing or not

File: IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs [hunk 3]
```diff
@@ -37,23 +40,25 @@ public InfoViewModel()
...
+        PauseImage = "volume_off_24px.png";
     }
 
     private  async void PlayCommandExcute(object obj)
     {
-      var playMusic = obj as Switch;
-      if (playMusic.IsToggled)
+      if (!IsPlaying)
       {
         Preferences.Set("playMusic", true);
```
Comment (saamerm) [inline IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs:51]: Instead of using preferences, use a property inside the 'AudioService' that I mentioned above

File: IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs [hunk 4]
```diff
@@ -71,11 +76,10 @@ private async void CheckusCommandExcute(object obj)
         // An unexpected error occured. No browser may be installed on the device.
       }
     }
-    private async void Play()
+    public static async void Play()
```
Comment (saamerm) [inline IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs:79]: Please don't make this static

File: IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs [hunk 5]
```diff
@@ -71,11 +76,10 @@ private async void CheckusCommandExcute(object obj)
...
     {
-      await CrossMediaManager.Current.Play("https://ia800806.us.archive.org/15/items/Mp3Playlist_555/AaronNeville-CrazyLove.mp3");
-     // await mediaInfo.Play("https://drive.google.com/file/d/1MSnVohtx-GfvzJJSYycuOq8y22q4o42h/view");
-     // this link not work 
+      var audio =  CrossMediaManager.Current;
+      await audio.PlayFromAssembly("music.mp3", typeof(BaseViewModel).Assembly);
```
Comment (saamerm) [inline IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs:82]: Please try to use an audio service. Please let me know if it's difficult or not possible

**Review by saamerm — CHANGES_REQUESTED** [review]

File: IslandLanding/IslandLanding/Converters/ImageConverter.cs (hunk 1 above)
Comment (amiraelsayedbadwy) [reply IslandLanding/IslandLanding/Converters/ImageConverter.cs:19]: okay

File: IslandLanding/IslandLanding/ViewModel/GamerTagViewModel.cs (hunk 2 above)
Comment (amiraelsayedbadwy) [reply IslandLanding/IslandLanding/ViewModel/GamerTagViewModel.cs:32]: i stored the state in the prefrences so i can check it ,

File: IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs (hunk 3 above)
Comment (amiraelsayedbadwy) [reply IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs:51]: ok

File: IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs (hunk 4 above)
Comment (amiraelsayedbadwy) [reply IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs:79]: ok

File: IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs (hunk 5 above)
Comment (amiraelsayedbadwy) [reply IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs:82]: your concern here about the nuget or what i dont understand  your point

File: IslandLanding/IslandLanding.Android/MainActivity.cs [hunk 6]
```diff
@@ -25,6 +25,7 @@ protected override void OnCreate(Bundle savedInstanceState)
       global::Xamarin.Forms.Forms.Init(this, savedInstanceState);
       Rg.Plugins.Popup.Popup.Init(this);
       CrossMediaManager.Current.Init();
+      var client = new HttpClient();
```
Comment (saamerm) [inline IslandLanding/IslandLanding.Android/MainActivity.cs]: @amiraelsayedbadwy why did you add this?

File: IslandLanding/IslandLanding/App.xaml [hunk 7]
```diff
@@ -9,6 +9,7 @@
       <Color x:Key="bordercolor">#4F3824</Color>
       <Color x:Key="orange">#E8A24F</Color>
       <Color x:Key="orangeLabel">#DDB967</Color>
+      <converters:ImageConverter x:Key="ImageConverter" />
```
Comment (saamerm) [inline IslandLanding/IslandLanding/App.xaml:12]: Are these changes needed ?

File: IslandLanding/IslandLanding/App.xaml [hunk 8]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8" ?>
 <Application xmlns="http://xamarin.com/schemas/2014/forms"
-             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml" xmlns:converters="clr-namespace:IslandLanding.Converters"
```
Comment (saamerm) [inline IslandLanding/IslandLanding/App.xaml:3]: Are these changes needed ?

**Review by saamerm — APPROVED** [review]
